circle.yml

```diff
@@ -10,6 +12,10 @@ general:
 test:
   post:
     - cat ./coverage/lcov.info | ./node_modules/.bin/coveralls --verbose
+    - >
+      if [ -n "${RUN_UPDATE}" ]; then
```

Comment:
What is the usage of that parameter? How am I supposed to use the `trigger_circleci_update.sh` script? Why not simply always update on merges to `master`?